ginivision/src/main/java/net/gini/android/vision/document/GiniVisionDocumentError.java

```diff
@@ -0,0 +1,51 @@
+package net.gini.android.vision.document;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+/**
+ * Created by Alpar Szotyori on 15.03.2018.
+ *
+ * Copyright (c) 2018 Gini GmbH.
+ */
+
+public class GiniVisionDocumentError implements Parcelable {
```

Comment:
Is this a real Error class or just a string wrapper? Since it there could be different error cases, would it make sense to have this as an `enum`?

Reply:
I used a class to have flexibility considering the error payload. Now it's only a string and when adding networking there might be a need for some additional information. It can also be that we use a String or an Enum.